packages/expo-build-properties/src/pluginConfig.ts

```diff
@@ -379,9 +379,10 @@ export interface PluginConfigTypeIos {
   privacyManifestAggregationEnabled?: boolean;
 
   /**
-   * Enables support for precompiled React Native iOS dependencies (`ReactNativeDependencies.xcframework`).
-   * This feature is available from React Native 0.80 and later when using the new architecture.
-   * From React Native 0.81, this setting will also use a precompiled React Native Core (`React.xcframework`).
+   * When set to `true`, React Native will be built from source.
```

Comment:
```suggestion
   * When set to `true`, React Native will be built from source.
```

Doesn't this line "Enables support for precompiled React Native iOS dependencies (`ReactNativeDependencies.xcframework`)."  define what this property does and it should be in the updated version as well?

Reading the updated version does not inform me of the purpose of this property.

Reply:
True 👍 

Thanks for pointing this out